packages/kbn-test/package.json

```diff
@@ -25,6 +26,7 @@
     "getopts": "^2.2.4",
     "glob": "^7.1.2",
     "parse-link-header": "^1.0.1",
+    "puppeteer": "3.1.0",
```

Comment:
Does it mean that Kibana installs 2 Chromium binaries from now on?

Reply:
Yep. Kibana reporting is using the old version and it is a risk introducing regression while bumping it